inc/classes/class-astra-import-export-loader.php

```diff
@@ -176,7 +176,11 @@ public static function import() {
 			// Delete existing dynamic CSS cache.
 			delete_option( 'astra-settings' );
```

Comment:
@patilvikasj I think we can delete Color palette's and typography preset's option too along with `astra-settings` ? Please check.

Reply:
@premanshup We are anyway updating those options using `update_option` so no need to delete them.